acceptance_tests/README.md

```diff
@@ -0,0 +1,85 @@
+# Helm Acceptance Tests
+
+This directory contains the source for Helm acceptance tests.
+
+The tests are written using [Robot Framework](https://robotframework.org/).
+
+## System requirements
+
+The following tools/commands are expected to be present on the base system
+prior to running the tests:
+
+- [kind](https://kind.sigs.k8s.io/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [python3](https://www.python.org/downloads/)
+- [pip](https://pip.pypa.io/en/stable/installing/)
+- [virtualenv](https://virtualenv.pypa.io/en/latest/installation/)
+
+## Running the tests
+
+From the root of this repo, run the following:
+
+```
+make acceptance
+```
+
+## Viewing the results
+
+Robot creates an HTML test report describing test successes/failures.
+
+To view the report, runt the following:
+
+```
+open .acceptance/report.html
```

Comment:
`open` is a command that only exists for MacOS. Can we instead recommend others to open this file in a browser?